Misc/NEWS.d/next/Library/2021-12-23-12-53-34.bpo-46162.bE3xrz.rst

```diff
@@ -0,0 +1 @@
+Make :class:`property` generic.
```

Comment:
_generic_ can be a bit vague (or generic 😉), could you rephrase this news snippet and the PR title to clarify this is about typing generic alias?